at_response.c

```diff
@@ -1266,7 +1266,7 @@ static int at_response_cmgr (struct pvt* pvt, const char * str, size_t len)
 	char		msg[4096];
 	int		res;
 	char		text_base64[40800];
-	size_t		msg_len;
+	size_t		msg_len = 4096;
```

Comment:
4096 still apears twice.

Either #define it or do:
```
char msg[4096];
size_t msg_len = sizeof(msg);
```

Reply:
Thanks for your review, removed magic 4096. 😄😄 @wdoekes